2020-09-27 22:04:26,029 DEBUG   MainThread:4631 [wandb_config.py:_load_defaults():154] no defaults not found in config-defaults.yaml
2020-09-27 22:04:26,097 DEBUG   MainThread:4631 [meta.py:_setup_code_git():49] probe for git information
2020-09-27 22:04:26,116 DEBUG   MainThread:4631 [run_manager.py:__init__():547] Initialized sync for rlmp/1bp6a0nq
2020-09-27 22:04:26,119 INFO    MainThread:4631 [run_manager.py:wrap_existing_process():1149] wrapping existing process 4619
2020-09-27 22:04:26,120 WARNING MainThread:4631 [io_wrap.py:register():108] SIGWINCH handler was not None: <Handlers.SIG_DFL: 0>
2020-09-27 22:04:26,145 INFO    MainThread:4631 [run_manager.py:init_run():928] system metrics and metadata threads started
2020-09-27 22:04:26,145 INFO    MainThread:4631 [run_manager.py:init_run():967] upserting run before process can begin, waiting at most 10 seconds
2020-09-27 22:04:26,239 INFO    Thread-15 :4631 [run_manager.py:_upsert_run():1056] saving pip packages
2020-09-27 22:04:26,239 INFO    Thread-15 :4631 [run_manager.py:_upsert_run():1058] initializing streaming files api
2020-09-27 22:04:26,240 INFO    Thread-15 :4631 [run_manager.py:_upsert_run():1065] unblocking file change observer, beginning sync with W&B servers
2020-09-27 22:04:26,240 INFO    MainThread:4631 [run_manager.py:wrap_existing_process():1166] informing user process we are ready to proceed
2020-09-27 22:04:26,243 INFO    MainThread:4631 [run_manager.py:_sync_etc():1298] entering loop for messages from user process
2020-09-27 22:04:27,095 INFO    Thread-4  :4631 [run_manager.py:_on_file_modified():692] file/dir modified: /home/ubuntu/rlmp/refactor/wandb/run-20200927_220425-1bp6a0nq/config.yaml
2020-09-27 22:04:27,193 INFO    Thread-4  :4631 [run_manager.py:_on_file_created():681] file/dir created: /home/ubuntu/rlmp/refactor/wandb/run-20200927_220425-1bp6a0nq/requirements.txt
2020-09-27 22:04:27,193 INFO    Thread-4  :4631 [run_manager.py:_on_file_created():681] file/dir created: /home/ubuntu/rlmp/refactor/wandb/run-20200927_220425-1bp6a0nq/output.log
2020-09-27 22:04:27,194 INFO    Thread-4  :4631 [run_manager.py:_on_file_created():681] file/dir created: /home/ubuntu/rlmp/refactor/wandb/run-20200927_220425-1bp6a0nq/wandb-metadata.json
2020-09-27 22:04:28,851 INFO    MainThread:4631 [run_manager.py:_sync_etc():1321] received message from user process: {"exitcode": 1}
2020-09-27 22:04:28,852 INFO    MainThread:4631 [run_manager.py:_sync_etc():1408] closing log streams and sending exitcode to W&B
2020-09-27 22:04:28,852 INFO    MainThread:4631 [run_manager.py:shutdown():1072] shutting down system stats and metadata service
2020-09-27 22:04:29,095 INFO    Thread-4  :4631 [run_manager.py:_on_file_modified():692] file/dir modified: /home/ubuntu/rlmp/refactor/wandb/run-20200927_220425-1bp6a0nq/output.log
2020-09-27 22:04:29,095 INFO    Thread-4  :4631 [run_manager.py:_on_file_created():681] file/dir created: /home/ubuntu/rlmp/refactor/wandb/run-20200927_220425-1bp6a0nq/wandb-events.jsonl
2020-09-27 22:04:29,148 INFO    MainThread:4631 [run_manager.py:shutdown():1086] stopping streaming files and file change observer
2020-09-27 22:04:30,095 INFO    Thread-4  :4631 [run_manager.py:_on_file_modified():692] file/dir modified: /home/ubuntu/rlmp/refactor/wandb/run-20200927_220425-1bp6a0nq/wandb-metadata.json
2020-09-27 22:04:32,150 INFO    MainThread:4631 [run_manager.py:_sync_etc():1420] process only ran for 6 seconds, not syncing files
FO    Thread-4  :4631 [1bp6a0nq:run_manager.py:_on_file_modified():692] file/dir modified: /home/ubuntu/rlmp/refactor/wandb/run-20200927_220425-1bp6a0nq/wandb-metadata.json
2020-09-27 22:04:32,150 INFO    MainThread:4631 [1bp6a0nq:run_manager.py:_sync_etc():1420] process only ran for 6 seconds, not syncing files
